app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/MainViewModel.kt

```diff
@@ -32,54 +32,54 @@ class MainViewModel : ViewModel() {
+            val gDeathPerMillion: ArrayList<String>? = gdp.getGlobalCovidData("deaths_per_1m_population")
+            val gStatistics: ArrayList<String>? = gdp.getGlobalCovidData("statistic_taken_at")
             delay(2000)
 
 
             /**
              * Set fetched data to DTO objects
              */
-            var globalDataDto = GlobalData(gCases!!.get(0), gActive!!.get(0),
+            val globalDataDto = GlobalData(gCases!!.get(0), gActive!!.get(0),
                 gTotalDeath!!.get(0), gRecovered!!.get(0), gNewCases!!.get(0),
                 gNewDeaths!!.get(0), gSeriousCritical!!.get(0), gCasesPerMillion!!.get(0),
                 gDeathPerMillion!!.get(0), gStatistics!!.get(0))
 
             gmData.value = globalDataDto
 
-            var cbc : CasesByCountryDataProvider = CasesByCountryDataProvider()
+            val cbc : CasesByCountryDataProvider = CasesByCountryDataProvider()
 
             /**
              * Call API to retrieve cases by country data
              */
-            var country: ArrayList<String>? = cbc.getCasesByCountryData("country_name")
-            var cases: ArrayList<String>? = cbc.getCasesByCountryData("cases")
-            var deaths: ArrayList<String>? = cbc.getCasesByCountryData("deaths")
-            var active: ArrayList<String>? = cbc.getCasesByCountryData("active_cases")
-            var totalRecovered: ArrayList<String>? = cbc.getCasesByCountryData("total_recovered")
-            var newDeaths: ArrayList<String>? = cbc.getCasesByCountryData("new_deaths")
-            var newCases: ArrayList<String>? = cbc.getCasesByCountryData("new_cases")
-            var seriousCritical: ArrayList<String>? = cbc.getCasesByCountryData("serious_critical")
-            var totalCasesPerMillionPopulation: ArrayList<String>? = cbc.getCasesByCountryData("total_cases_per_1m_population")
+            val country: ArrayList<String>? = cbc.getCasesByCountryData("country_name")
+            val cases: ArrayList<String>? = cbc.getCasesByCountryData("cases")
+            val deaths: ArrayList<String>? = cbc.getCasesByCountryData("deaths")
+            val active: ArrayList<String>? = cbc.getCasesByCountryData("active_cases")
+            val totalRecovered: ArrayList<String>? = cbc.getCasesByCountryData("total_recovered")
+            val newDeaths: ArrayList<String>? = cbc.getCasesByCountryData("new_deaths")
+            val newCases: ArrayList<String>? = cbc.getCasesByCountryData("new_cases")
+            val seriousCritical: ArrayList<String>? = cbc.getCasesByCountryData("serious_critical")
+            val totalCasesPerMillionPopulation: ArrayList<String>? = cbc.getCasesByCountryData("total_cases_per_1m_population")
             delay(2000)
 
             /**
              * Set fetched country data to DTO object
              */
-            var casesDto = Cases(country!!, cases!!, deaths!!, active!!, totalRecovered!!, newDeaths!!,
```

Comment:
Changed as the variables remained constant so the variable would be better off as a val